databricks_cli/stack/cli.py

```diff
@@ -34,16 +34,19 @@
 @click.command(context_settings=CONTEXT_SETTINGS,
                short_help='Deploy stack given a JSON configuration of the stack')
 @click.argument('config_path', type=click.Path(exists=True), required=True)
+@click.option('--overwrite-notebooks', '-o', is_flag=True, help='Include to overwrite existing'
+                                                                'notebooks in the workspace.')
 @profile_option
 @eat_exceptions
 @provide_api_client
-def deploy(api_client, config_path):
+def deploy(api_client, config_path, **kwargs):
     """
     Deploy a stack to the databricks workspace given a JSON stack configuration template.
     """
     click.echo('#' * 80)
     click.echo('Deploying stack at: ' + config_path)
-    StackApi(api_client).deploy(config_path)
+    click.echo(kwargs)
```

Comment:
Can we combine this with the previous line?
` click.echo('Deploying stack at: {}, with options {}'.format(config_path, kwargs))`

Reply:
I actually just did this echo for debugging lol but that's a good idea, I'll add that.